components/message.py

```diff
@@ -20,15 +28,18 @@ class Message(Component):
     def draw_base(self, canvas: Canvas):
         pass
 
+    @abstractmethod
     def draw_data(self, canvas: Canvas, data: Data):
-        if not data.has_message():
-            return
-        message = data.get_message()
-        display_str = f"Message: {message}"
+        pass
```

Comment:
Following from https://github.com/monash-human-power/raspicam/pull/48#discussion_r491236379, we don't actually need this method specified here at all, the abstract method will be inherited from `Component`.